connectivity/check/context.go

```diff
@@ -289,6 +295,17 @@ func (ct *ConnectivityTest) report() error {
 
 		return fmt.Errorf("%d tests failed", nf)
 	}
+	// Report Performance results
+	if len(ct.PerfResults) > 0 {
```

Comment:
Nit: this check shouldn't be needed, the `for` loop below will not run for an empty `ct.PerfResults`.

Reply:
habit... damnit python brain.